decompiler/config/jak2/all-types.gc

```diff
@@ -29661,11 +29664,11 @@
   :size-assert         #x10
   :flag-assert         #xb00000010
   (:methods
-    (idle-control-method-9 () none 9)
-    (idle-control-method-10 () none 10)
+    (idle-control-method-9 (_type_ idle-control-frame idle-control-frame) idle-control 9)
+    (idle-control-method-10 (_type_ uint32 uint32) none 10)
```

Comment:
use register semantics types such as `uint` (and `int`) instead of memory semantics types.